nixos/modules/services/backup/crashplan-small-business.nix

```diff
@@ -48,7 +48,7 @@ with lib;
       description = "CrashPlan Backup Engine";
 
       wantedBy = [ "multi-user.target" ];
-      after    = [ "network.target" "local-fs.target" ];
+      after    = [ "network.target" ];
```

Comment:
You can drop this file as it won't be an issue [much longer](https://github.com/NixOS/nixpkgs/pull/67818).